packages/office-addin-manifest/src/export.ts

```diff
@@ -0,0 +1,36 @@
+import * as fs from "fs";
+import * as fsExtra from "fs-extra";
+import * as jszip from "jszip";
+import * as path from "path";
+
+export function exportMetadataPackage(
+  output: string = "package/manifest.zip",
+  manifest: string = "manifest.json",
+  assets: string = "assets"
+): string {
+  const zip = new jszip();
+
+  if (fs.existsSync(manifest)) {
+    const manifestData = fs.readFileSync(manifest);
+    zip.file("manifest.json", manifestData);
+  } else {
+    throw new Error(`The file '${manifest}' does not exist`);
+  }
+
+  if (fs.existsSync(assets)) {
+    const files: string[] = fs.readdirSync(assets);
+    zip.folder(assets);
+    files.forEach((element) => {
+      const filePath = assets + "/" + element;
```

Comment:
Can we use a library to make this pathing work? There are probably some already made.
I remember for something like `path.join()`.
I say that because using a library would make this work for other Operation Systems as well.

Reply:
I didn't think the notation with '+' depended on the OS . . . but using the library makes better sense anyway.

Reply:
I'm now completely sure I won't work on other OS, it is just sometimes the OS uses `//` or other ways to create paths, which is different than `/`.